utils/src/ooinstall/cli_installer.py

```diff
@@ -261,23 +307,32 @@ def check_hosts_config(oo_cfg):
             if master_lb[0].master or master_lb[0].node:
                 click.echo('The Master load balancer is configured as a master or node. Please correct this.')
                 sys.exit(0)
+            # Check for another host with same connect_to?
         else:
             message = """
-No HAProxy given in config. Either specify one or provide a load balancing solution
-of your choice to balance the master API (port 8443) on all master hosts.
+WARNING: No master load balancer specified in config. If you proceed you will
+need to provide a load balancing solution of your choice to balance the
+API (port 8443) on all master hosts.
```

Comment:
@brenton The tricky part is that the openshift_master_cluster_hostname is the one that is used to put in the node configs for the node to contact the master in the case of an HA deployment, we wouldn't necessarily want that traffic traversing an external security zone.